Makefile

```diff
@@ -10,6 +10,12 @@ ifeq ($(VERSION_FULL),)
 VERSION_FULL := $(VERSION)
 endif
 
+WWROOT ?= /var/lib
```

Comment:
@jcsiadal In my system before this change the directory where chroots and overlays are stored is `/var/warewulf` **not** `/var/lib/warewulf`. See in my comment above the subsection **reinstall v4.2.0**. It might be fine to move this to any directory that admin wished at build time but I think the `WWROOT` directory should be kept as the current default, which is `/var`, otherwise this may be a breaking change for an upgrade where `WWROOT="/var" make` was not used to build.

Reply:
@jcsiadal What was your perspective on this?

Reply:
My thought is that because it can be moved anywhere, our default installation should be FHS-compliant. If an admin builds and installs from source as-is, it shouldn't do anything that's not allowed under LSB or FHS. The past is a sunk cost.

I originally had it moved under /srv, but the comment by @elguero really changed my mind. It's the third comment, but also note the one after.